db/seeds.rb

```diff
@@ -1,7 +1,50 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'csv'
+require 'faker'
+
+# add cities in DB
+
+City.all.destroy_all
```

Comment:
```suggestion
City.find_each(&:destroy!)
```
In this case (and in most case) it's best to use the `!` version, otherwise it may silently fail, and you'll never know. Same for the `create` below